src/tests/tests.spec.js

```diff
@@ -0,0 +1,108 @@
+import 'dotenv/config';
+
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+
+test.describe('filtering checks', () => {
+    test.beforeEach(async ({ loginPage }) => {
+        const userName = process.env.STANDARD_USER_NAME;
+        const password = process.env.PASSWORD;
+        await loginPage.navigate();
+        await loginPage.performLogin(userName, password);
+    });
+
+    test('Perform and verify sorting on the Inventory page (az)', async ({ inventoryPage }) => {
+        const defaultList = await inventoryPage.getInventoryItemsList();
+        await inventoryPage.performSorting('az');
+        const sortedList = await inventoryPage.getInventoryItemsList();
+        expect(sortedList).toEqual(defaultList.sort((a, b) => a.name.localeCompare(b.name)));
+    });
+
+    test('Perform and verify sorting on the Inventory page (za)', async ({ inventoryPage }) => {
+        const defaultList = await inventoryPage.getInventoryItemsList();
+        await inventoryPage.performSorting('za');
+        const sortedList = await inventoryPage.getInventoryItemsList();
+        expect(sortedList).toEqual(defaultList.sort((a, b) => b.name.localeCompare(a.name)));
+    });
+
+    test('Perform and verify sorting on the Inventory page (hilo)', async ({ inventoryPage }) => {
+        const defaultList = await inventoryPage.getInventoryItemsList();
+        await inventoryPage.performSorting('hilo');
+        const sortedList = await inventoryPage.getInventoryItemsList();
+        expect(sortedList).toEqual(defaultList.sort((a, b) => b.price - a.price));
+    });
+
+    test('Perform and verify sorting on the Inventory page (lohi)', async ({ inventoryPage }) => {
+        const defaultList = await inventoryPage.getInventoryItemsList();
+        await inventoryPage.performSorting('lohi');
+        const sortedList = await inventoryPage.getInventoryItemsList();
+        expect(sortedList).toEqual(defaultList.sort((a, b) => a.price - b.price));
+    });
```

Comment:
This could be updated with parametrized test. You could find example [here](https://github.com/hlazkov/unitedcode-taf/blob/main/tests/parfumFilters.spec.ts). Its a good way to reuse code. Every time you find yourself writing similar code several times its a good idea to try to reuse it!